easybuild/easyconfigs/r/re2c/re2c-3.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 're2c'
+version = '3.1'
+
+homepage = 'https://re2c.org'
+description = """re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
+fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using
+traditional table-driven approach, re2c encodes the generated finite state automata directly in the form
+of conditional jumps and comparisons."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/skvadrik/re2c/releases/download/%(version)s']
+sources = ['%(name)s-%(version)s.tar.xz']
+checksums = ['0ac299ad359e3f512b06a99397d025cfff81d3be34464ded0656f8a96676c029']
+
+builddependencies = [('binutils', '2.40')]
```

On generoso the build failed with:
```
configure: error: no suitable Python interpreter found
```
```suggestion
builddependencies = [
    ('binutils', '2.40'),
    ('Python', '3.11.3')
]
```

---

thanks for fixing